org.eclipse.egit.ui/src/org/eclipse/egit/ui/internal/preferences/SynchronizePreferencePage.java

```diff
@@ -99,8 +99,14 @@ protected void initialize() {
 		org.eclipse.egit.core.Activator coreActivator = org.eclipse.egit.core.Activator
 				.getDefault();
 		List<String[]> strategies = new ArrayList<>();
+
+		// We need to add a default name for EGit default strategy in order to
+		// be able to distinguish a state where user have not choose any
```

Comment:
I don't agree: Actually, using the option -preferenceCustomization in the eclipse.ini file defines the default values of the properties in the referenced file.
This means that these values should be the values taken when the user has not chosen anything.
Plus, the value should not point to an undefined key of merge strategy, as defined by JGit.